### ISSUE TITLE
cannot replace "latest" on each lambda-layer ARN

### ISSUE THREAD

**Issue by hisato-kawaji** [mooyoul/serverless-latest-layer-version#5]
I prepare the code below but the plugin seems not to work.

Ex.

```
functions:
  SampleFunction:
    handler: sample.handler
    layers:
      - arn:aws:lambda:us-east-1:00000000:layer:SampleLayer:latest
```

when I try  to `sls deploy -v`, i got the warning message, and the serverless command failed as below.(similar to the issue https://github.com/mooyoul/serverless-latest-layer-version/issues/3 but the command is different)

```
Serverless: [serverless-latest-layer-version] Detected unknown Layer ARN arn:aws:lambda:us-east-1:00000000:layer:SampleLayer:latest. Please create a new issue to github.com/mooyoul/serverless-latest-layer-version

.
.
.

  Serverless Error ---------------------------------------
 
  An error occurred: SampleFunctionLambdaFunction - 1 validation error detected: Value '[arn:aws:lambda:us-east-1:00000000:layer:SampleLayer:latest]' at 'layers' failed to satisfy constraint: Member must satisfy constraint: [Member must have length less than or equal to 140, Member must have length greater than or equal to 1, Member must satisfy regular expression pattern: (arn:[a-zA-Z0-9-]+:lambda:[a-zA-Z0-9-]+:\d{12}:layer:[a-zA-Z0-9-_]+:[0-9]+)|(arn:[a-zA-Z0-9-]+:lambda:::awslayer:[a-zA-Z0-9-_]+)] (Service: AWSLambdaInternal; Status Code: 400; Error Code: ValidationException; Request ID: XXXXXXXXX).
```

Software
```
node: 10.16.3
Serverless Framework: 1.60.0
serverless-latest-layer-version: 2.0.0
```

**Comment by mooyoul** [comment]
Interesting. I'm sure that's not caused by `-v` (verbose) option. Anyway. Does your layer name has any upper case characters?

**Comment by hisato-kawaji** [comment]
@mooyoul 

>  Does your layer name has any upper case characters?

Yes, the layer name includes upper case characters.

**Comment by hisato-kawaji** [comment]
If the layer does not exist, the debug log shows that
```
An error occurred: CareUsersPOSTLambdaFunction - Layer version arn:aws:lambda:us-east-1:00000000:layer:SampleLayer:999 does not exist. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: XXXXXXXXX).
```

**Comment by mooyoul** [comment]
Confirmed. There was a business logic that converts layer ARN to lower case, and it caused this bug. Fixed version will be released on next day.

**Comment by mooyoul** [comment]
I just published v2.1.0 which fixes this issue. Please try again with new version and let me know the results. Thanks for your patience! Happy new year!
